pandas/core/algorithms.py

```diff
@@ -1864,7 +1860,7 @@ def searchsorted(arr, value, side="left", sorter=None):
         else:
             value = array(value, dtype=dtype)
     elif not (
-        is_object_dtype(arr) or is_numeric_dtype(arr) or is_categorical_dtype(arr)
+            is_object_dtype(arr) or is_numeric_dtype(arr) or is_categorical_dtype(arr)
```

Comment:
can you not add unrelated changes (all of this whitespace)

Reply:
Sorry for that.